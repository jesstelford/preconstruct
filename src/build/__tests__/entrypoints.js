// @flow
import build from "../";
import fixturez from "fixturez";
import { initBasic, snapshotDistFiles } from "../../../test-utils";

jest.mock("../../prompt");

const f = fixturez(__dirname);

test("non default src", async () => {
  let tmpPath = f.copy("non-default-src");
  await initBasic(tmpPath);
  await build(tmpPath);
  await snapshotDistFiles(tmpPath);
});
