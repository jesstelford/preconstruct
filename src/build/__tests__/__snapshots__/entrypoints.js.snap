// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`monorepo: non-default-src.cjs.dev.js 1`] = `
"'use strict';

Object.defineProperty(exports, '__esModule', {
  value: true
});
let wow = \\"something\\";
exports.wow = wow;
"
`;

exports[`monorepo: non-default-src.cjs.js 1`] = `
"'use strict';

if (process.env.NODE_ENV === \\"production\\") {
  module.exports = require(\\"./non-default-src.cjs.prod.js\\");
} else {
  module.exports = require(\\"./non-default-src.cjs.dev.js\\");
}
"
`;

exports[`monorepo: non-default-src.cjs.prod.js 1`] = `
"\\"use strict\\";
Object.defineProperty(exports, \\"__esModule\\", { value: !0 });
let wow = \\"something\\";
exports.wow = wow;
"
`;

exports[`monorepo: non-default-src.esm.js 1`] = `
"let wow = \\"something\\";

export { wow };
"
`;
